--- sys/synthesis/submodules/sys_mm_interconnect_0.v	2023-10-29 17:53:11.806522733 +0200
+++ sys/synthesis/submodules/sys_mm_interconnect_0.v	2023-10-29 18:04:50.827992821 +0200
@@ -3105,7 +3105,7 @@
 		.USE_PACKETS         (1),
 		.USE_FILL_LEVEL      (0),
 		.EMPTY_LATENCY       (1),
-		.USE_MEMORY_BLOCKS   (0),
+		.USE_MEMORY_BLOCKS   (1),
 		.USE_STORE_FORWARD   (0),
 		.USE_ALMOST_FULL_IF  (0),
 		.USE_ALMOST_EMPTY_IF (0)
@@ -4271,7 +4271,7 @@
 		.USE_PACKETS         (1),
 		.USE_FILL_LEVEL      (0),
 		.EMPTY_LATENCY       (1),
-		.USE_MEMORY_BLOCKS   (0),
+		.USE_MEMORY_BLOCKS   (1),
 		.USE_STORE_FORWARD   (0),
 		.USE_ALMOST_FULL_IF  (0),
 		.USE_ALMOST_EMPTY_IF (0)
@@ -4437,7 +4437,7 @@
 		.USE_PACKETS         (1),
 		.USE_FILL_LEVEL      (0),
 		.EMPTY_LATENCY       (1),
-		.USE_MEMORY_BLOCKS   (0),
+		.USE_MEMORY_BLOCKS   (1),
 		.USE_STORE_FORWARD   (0),
 		.USE_ALMOST_FULL_IF  (0),
 		.USE_ALMOST_EMPTY_IF (0)
--- sys/synthesis/submodules/sys_mm_interconnect_1.v	2023-10-29 17:53:12.338526762 +0200
+++ sys/synthesis/submodules/sys_mm_interconnect_1.v	2023-10-29 18:05:17.996198049 +0200
@@ -2112,7 +2112,7 @@
 		.USE_PACKETS         (1),
 		.USE_FILL_LEVEL      (0),
 		.EMPTY_LATENCY       (1),
-		.USE_MEMORY_BLOCKS   (0),
+		.USE_MEMORY_BLOCKS   (1),
 		.USE_STORE_FORWARD   (0),
 		.USE_ALMOST_FULL_IF  (0),
 		.USE_ALMOST_EMPTY_IF (0)
